tests/e2e/helper_test.go

```diff
@@ -115,27 +117,27 @@ func setupDSCInstance(name string) *dscv1.DataScienceCluster {
 				// keep dashboard as enabled, because other test is rely on this
 				Dashboard: componentApi.DSCDashboard{
 					ManagementSpec: common.ManagementSpec{
-						ManagementState: operatorv1.Managed,
+						ManagementState: operatorv1.Removed,
```

Comment:
I guess this switch from Managed to Removed (and the subsequent ones below) is temporary and won't be merged like this?

Reply:
That's a good question :)

The reason I changed them is:
- when running the individual component tests, having all the components installed causes a lot of noise and resource consumption
- I think it should be better to have each test to set the context, instead of making assumption so a test would work as expected, regardless of any previous state (i.e. when running the test in a debugger, you don't want the test to fail because on a previous iteration, a component were set to `Removed`)

That said, I don't have any strong opinion so I can revert it before merging.